### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Vasyl-Poremchuk**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,47 @@
...
+        print("The hunt began!")
+
+
+class Dog(Animal):
+    def __init__(self, name: str,
+                 is_hungry: bool = True
+                 ) -> None:
+        super().__init__(name, 7, is_hungry)
+        self.is_hungry = is_hungry
+
```
Comment (StasOkhrym) [inline app/main.py:40]: and here.
Everting else looks great.

**Review by StasOkhrym — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 1 above)
Comment (Vasyl-Poremchuk) [reply app/main.py:37]: Done!

**Review by StasOkhrym — APPROVED** [review]
Looks great!

**Review by Nattalli — APPROVED** [review]
👍🏻

**Pull request closed without merging**